go.mod

```diff
@@ -57,19 +57,25 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require github.com/willabides/kongplete v0.3.0
+
```

Comment:
Did you add it using `go get`?  The fact that it's in a separate block is strange.

Reply:
I think I did, should I move it into the first block instead?